src/ui/webeiddialog.cpp

```diff
@@ -358,10 +358,7 @@ void WebEidDialog::setupPinPadProgressBarAndEmitWait()
     okButton->hide();
     cancelButton->hide();
     ui->pinEntryTimeoutProgressBar->show();
-
-    // FIXME: translation
-    const auto text = ui->pinTitleLabel->text().replace(':', QStringLiteral(" using PIN-pad"));
-    ui->pinTitleLabel->setText(text);
+    ui->pinTitleLabel->setText(tr("Please enter authentication PIN (PIN 1) using PIN-pad"));
```

Comment:
This label is used both during authentication and signing, what if we solve this as follows?

```suggestion
    ui->pinTitleLabel->setText(tr("Please enter %s using PIN pad")
                                   .arg(currentCommand == CommandType::AUTHENTICATE
                                            ? tr("authentication PIN (PIN 1)")
                                            : tr("signing PIN (PIN 2)")));
```